apps/camera/style/index.css

```diff
@@ -4,7 +4,7 @@ body {
   height: 100%;
   margin: 0;
   padding: 0;
-  font-family: "Open Sans", arial, "sans serif";
+  font-family: 'Readex Pro', sans-serif;
```

Comment:
please use double quotes for strings litteral

Reply:
alright